shared/util/open-url.desktop.js

```diff
@@ -1,5 +1,9 @@
 import {shell} from 'electron'
 
 export default function openURL (url) {
+  if (!url) {
```

Comment:
I made a similar change, but one problem with this is when we preformat the url (like we do with twitter).

In those cases the url will never be null or '' and we'll fail.